src/user_management/user.resolver.js

```diff
@@ -1,9 +1,36 @@
+import { AuthenticationError } from 'apollo-server-express'
+
 import models from '~/src/service_providers/sequelize/models'
 
 export default {
+  Query: {
+    users: (_,args) => {
+      return models.user.findAll()
+    },
+    currentUser: {
+
+    },
+    logInUser: async (_,{email, password}) => {
+      const user = await models.user.findOne({
+        where: {
+          email
+        }
+      }) 
+      if (user && await user.validatePass(password)){
+        return user
+      }
+      throw new AuthenticationError('Invalid credentials')
+    }
+  },
   Mutation: {
-    createUser: () => {
-      return models.user.create()
+    createUser: async (_,{input}) => {
+      const response = await models.user.create({
+        firstName: input.firstName,
+        lastName: input.lastName,
+        email: input.email,
+        password: input.password
+      })
+      return response
     }
```

Comment:
The GraphQL schema is expecting something different than what you're returning here. You'll need to return an object matching the shape required on the schema:
```suggestion
    createUser: async (_, { input }) => {
      const user = await models.user.create({
        firstName: input.firstName,
        lastName: input.lastName,
        email: input.email,
        password: input.password
      })
      
      return {
        code: 'OK',
        success: true,
        message: 'User created successfully',
        user
      }
    }
```